data_structures/src/serializers.rs

```diff
@@ -104,7 +104,13 @@ struct VersionWitnetArgs {
     version: u32,
 }
 
-impl TryFrom<Vec<u8>> for Message {
+pub trait MyTryFrom<T>: Sized {
```

Comment:
```suggestion
pub trait TryFrom<T>: Sized {
```
`TryFrom` from `std` can be overlapped locally.

Reply:
True :+1: